src/components/modals/StakingModal.tsx

```diff
@@ -211,6 +179,10 @@ const StakingModal = ({
           queryKey: ['gateways'],
           refetchType: 'all',
         });
+        queryClient.invalidateQueries({
+          queryKey: ['balances'],
+          refetchType: 'all',
+        });
```

Comment:
```suggestion
        queryClient.invalidateQueries({
          queryKey: ['balances'],
          refetchType: 'all',
          exact: false
        });
```

Reply:
The default is false for exact, so we only need to add exact:true when we want that behavior